ui/components/app/cancel-speedup-popover/cancel-speedup-popover.js

```diff
@@ -1,12 +1,14 @@
+/* eslint-disable prettier/prettier */
```

Comment:
Same question here.

Reply:
Those lines were just to disable prettier formatting in vscode, I will remove those lines and commit changes once I get a response from @NidhiKJha

Reply:
Hey @georgewrmarshall 
I have removed those lines and made changes as told by @NidhiKJha
Can you please review the code now?
Thanks!